<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1623</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="1" column="0">
     <widget class="QComboBox" name="comboBox">
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <item>
       <property name="text">
        <string>Белорусский</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Немецкий</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Французский</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Арабский</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Иврит</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Китайский</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="5" column="0">
     <layout class="QVBoxLayout" name="verticalLayout"/>
    </item>
    <item row="0" column="0" colspan="2">
     <widget class="QTextEdit" name="textEdit">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="tabChangesFocus">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="pushButton">
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="text">
       <string>Старт</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1" rowspan="4">
     <widget class="QGroupBox" name="groupBox">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="title">
       <string/>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="2" colspan="2">
        <widget class="QPushButton" name="pushButton3">
         <property name="text">
          <string>2</string>
         </property>
        </widget>
       </item>
       <item row="2" column="10" colspan="2">
        <widget class="QPushButton" name="pushButton30">
         <property name="text">
          <string>п</string>
         </property>
        </widget>
       </item>
       <item row="3" column="22" colspan="3">
        <widget class="QPushButton" name="pushButton44">
         <property name="text">
          <string>б</string>
         </property>
        </widget>
       </item>
       <item row="3" column="2" colspan="2">
        <widget class="QPushButton" name="pushButton37">
         <property name="text">
          <string>я</string>
         </property>
        </widget>
       </item>
       <item row="1" column="3" colspan="2">
        <widget class="QPushButton" name="pushButton15">
         <property name="text">
          <string>ц</string>
         </property>
        </widget>
       </item>
       <item row="0" column="4" colspan="2">
        <widget class="QPushButton" name="pushButton4">
         <property name="font">
          <font>
           <hintingpreference>PreferDefaultHinting</hintingpreference>
          </font>
         </property>
         <property name="text">
          <string>3</string>
         </property>
        </widget>
       </item>
       <item row="0" column="6" colspan="2">
        <widget class="QPushButton" name="pushButton5">
         <property name="text">
          <string>4</string>
         </property>
        </widget>
       </item>
       <item row="3" column="18" colspan="3">
        <widget class="QPushButton" name="pushButton43">
         <property name="text">
          <string>ь</string>
         </property>
        </widget>
       </item>
       <item row="0" column="13" colspan="3">
        <widget class="QPushButton" name="pushButton8">
         <property name="text">
          <string>7</string>
         </property>
        </widget>
       </item>
       <item row="2" column="20" colspan="3">
        <widget class="QPushButton" name="pushButton33">
         <property name="text">
          <string>л</string>
         </property>
        </widget>
       </item>
       <item row="3" column="26" colspan="3">
        <widget class="QPushButton" name="pushButton45">
         <property name="text">
          <string>ю</string>
         </property>
        </widget>
       </item>
       <item row="0" column="21" colspan="3">
        <widget class="QPushButton" name="pushButton10">
         <property name="text">
          <string>9</string>
         </property>
        </widget>
       </item>
       <item row="0" column="25" colspan="3">
        <widget class="QPushButton" name="pushButton11">
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item row="0" column="10" colspan="2">
        <widget class="QPushButton" name="pushButton7">
         <property name="text">
          <string>6</string>
         </property>
        </widget>
       </item>
       <item row="1" column="31" colspan="3">
        <widget class="QPushButton" name="pushButton24">
         <property name="text">
          <string>х</string>
         </property>
        </widget>
       </item>
       <item row="3" column="14" colspan="3">
        <widget class="QPushButton" name="pushButton42">
         <property name="text">
          <string>т</string>
         </property>
        </widget>
       </item>
       <item row="1" column="27" colspan="3">
        <widget class="QPushButton" name="pushButton23">
         <property name="text">
          <string>з</string>
         </property>
        </widget>
       </item>
       <item row="2" column="4">
        <widget class="QPushButton" name="pushButton27">
         <property name="text">
          <string>ы</string>
         </property>
        </widget>
       </item>
       <item row="2" column="8" colspan="2">
        <widget class="QPushButton" name="pushButton29">
         <property name="text">
          <string>а</string>
         </property>
        </widget>
       </item>
       <item row="2" column="24" colspan="3">
        <widget class="QPushButton" name="pushButton34">
         <property name="text">
          <string>д</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1" colspan="2">
        <widget class="QPushButton" name="pushButton14">
         <property name="text">
          <string>й</string>
         </property>
        </widget>
       </item>
       <item row="2" column="16" colspan="3">
        <widget class="QPushButton" name="pushButton32">
         <property name="text">
          <string>о</string>
         </property>
        </widget>
       </item>
       <item row="0" column="8" colspan="2">
        <widget class="QPushButton" name="pushButton6">
         <property name="text">
          <string>5</string>
         </property>
        </widget>
       </item>
       <item row="2" column="32" colspan="2">
        <widget class="QPushButton" name="pushButton36">
         <property name="text">
          <string>э</string>
         </property>
        </widget>
       </item>
       <item row="1" column="11" colspan="3">
        <widget class="QPushButton" name="pushButton19">
         <property name="text">
          <string>н</string>
         </property>
        </widget>
       </item>
       <item row="3" column="9">
        <widget class="QPushButton" name="pushButton40">
         <property name="text">
          <string>м</string>
         </property>
        </widget>
       </item>
       <item row="2" column="12" colspan="3">
        <widget class="QPushButton" name="pushButton31">
         <property name="text">
          <string>р</string>
         </property>
        </widget>
       </item>
       <item row="0" column="33" colspan="2">
        <widget class="QPushButton" name="pushButton13">
         <property name="text">
          <string>=</string>
         </property>
        </widget>
       </item>
       <item row="1" column="9" colspan="2">
        <widget class="QPushButton" name="pushButton18">
         <property name="text">
          <string>е</string>
         </property>
        </widget>
       </item>
       <item row="3" column="11" colspan="2">
        <widget class="QPushButton" name="pushButton41">
         <property name="text">
          <string>і</string>
         </property>
        </widget>
       </item>
       <item row="4" column="6" colspan="15">
        <widget class="QPushButton" name="pushButton47">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="1" column="34">
        <widget class="QPushButton" name="pushButton25">
         <property name="text">
          <string>'</string>
         </property>
        </widget>
       </item>
       <item row="3" column="6" colspan="2">
        <widget class="QPushButton" name="pushButton39">
         <property name="text">
          <string>с</string>
         </property>
        </widget>
       </item>
       <item row="0" column="17" colspan="3">
        <widget class="QPushButton" name="pushButton9">
         <property name="text">
          <string>8</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="pushButton2">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>1</string>
         </property>
        </widget>
       </item>
       <item row="0" column="29" colspan="3">
        <widget class="QPushButton" name="pushButton12">
         <property name="text">
          <string>-</string>
         </property>
        </widget>
       </item>
       <item row="1" column="5" colspan="2">
        <widget class="QPushButton" name="pushButton16">
         <property name="text">
          <string>у</string>
         </property>
        </widget>
       </item>
       <item row="2" column="6" colspan="2">
        <widget class="QPushButton" name="pushButton28">
         <property name="text">
          <string>в</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QPushButton" name="pushButton1">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>ё</string>
         </property>
        </widget>
       </item>
       <item row="3" column="4" colspan="2">
        <widget class="QPushButton" name="pushButton38">
         <property name="text">
          <string>ч</string>
         </property>
        </widget>
       </item>
       <item row="1" column="15" colspan="3">
        <widget class="QPushButton" name="pushButton20">
         <property name="text">
          <string>г</string>
         </property>
        </widget>
       </item>
       <item row="1" column="23" colspan="3">
        <widget class="QPushButton" name="pushButton22">
         <property name="text">
          <string>ў</string>
         </property>
        </widget>
       </item>
       <item row="1" column="19" colspan="3">
        <widget class="QPushButton" name="pushButton21">
         <property name="text">
          <string>ш</string>
         </property>
        </widget>
       </item>
       <item row="1" column="7" colspan="2">
        <widget class="QPushButton" name="pushButton17">
         <property name="text">
          <string>к</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QPushButton" name="pushButton26">
         <property name="text">
          <string>ф</string>
         </property>
        </widget>
       </item>
       <item row="2" column="28" colspan="3">
        <widget class="QPushButton" name="pushButton35">
         <property name="text">
          <string>ж</string>
         </property>
        </widget>
       </item>
       <item row="3" column="30" colspan="3">
        <widget class="QPushButton" name="pushButton46">
         <property name="text">
          <string>.</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="4" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_2"/>
    </item>
    <item row="3" column="0">
     <widget class="QPushButton" name="pushButton_2">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="text">
       <string>Финиш</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1623</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
